multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -272,7 +260,7 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	if err != nil {
```

Comment:
Question - why we check if Service or Endpoints changed in this func for reconciling ServiceExport? Service/Endpoints change should be handled by Service/Endpoints watcher handlers but not here?

Reply:
The reason is it's the same reconciling logic to update ResourceExport when there is any ServiceExport/Service/Endpoint update, if we split them into different handlers, then there will be duplicate codes. but I probably can revisit this part after we change the Endpoint to Service Cluster IP, maybe split them into two parts with some shared codes.

Reply:
Better to add some comments about this.

BTW, I feel we should either make objMapFunc a member of ServiceExportReconciler or rename it to make clear it is for Service/Endpoints to ServiceExport mapping.

Reply:
done